site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -188,17 +201,16 @@ Create a node pool for [Metrics]({{< relref "../../Guides/metrics.md" >}}) if yo
 ```bash
 gcloud container node-pools create agones-metrics \
   --cluster=[CLUSTER_NAME] \
-  --no-enable-autoupgrade \
+  --region=[COMPUTE_REGION] \
   --node-taints agones.dev/agones-metrics=true:NoExecute \
   --node-labels agones.dev/agones-metrics=true \
   --num-nodes=1 \
   --machine-type=e2-standard-4
 ```
+where `[CLUSTER_NAME]`is the name of the cluster you created and `[COMPUTE_REGION]` is the region.
 
 Flag explanations:
 
-* cluster: The name of your existing cluster in which the node pool is created.
-* no-enable-autoupgrade: Disable automatic upgrades for nodes to reduce the likelihood of in-use games being disrupted.
 * node-taints: The Kubernetes taints to automatically apply to nodes in this node pool.
 * node-labels: The Kubernetes labels to automatically apply to nodes in this node pool.
 * num-nodes: The Agones system controllers only require a single node of capacity to run. For faster recovery time in the event of a node failure, you can increase the size to 2.
```

Comment:
This looks like it was copy/pasted from a section that no longer exists. If you are using a regional cluster, you get 3 nodes (so the part about 1 vs 2 is incorrect) and this is an explanation for the metrics pool not the agones system pool.